src/components/selection_input/selection_input.ts

```diff
@@ -18,6 +19,12 @@ css/* scss */ `
         right: 7px;
         top: 4px;
       }
+      .o-drag-handle {
+        display: flex;
+        align-items: center;
+        margin-bottom: 0.5rem;
```

Comment:
nitpick: 0.25rem looks a bit better IMO.

In any case, you can use bootrap for this one, `mb-1 = 0.25rem, mb-2 = 0.5rem`